src/trading/tradingSdk.ts

```diff
@@ -12,12 +12,26 @@ import { getQuoteWithSigner } from './getQuote'
 import { postSellNativeCurrencyOrder } from './postSellNativeCurrencyOrder'
 import { getSigner, swapParamsToLimitOrderParams } from './utils'
 import { getPreSignTransaction } from './getPreSignTransaction'
+import { log } from './consts'
+import { OrderBookApi } from '../order-book'
+
+interface TradingSdkOptions {
+  logs: boolean
+  orderBookApi: OrderBookApi
+}
 
 export class TradingSdk {
-  constructor(public readonly traderParams: TraderParameters) {}
+  constructor(
+    public readonly traderParams: TraderParameters,
+    public readonly options: Partial<TradingSdkOptions> = { logs: false }
+  ) {
+    if (options.logs) {
+      log.enabled = true
```

Comment:
Its a bit strange pattern to mutate a global var like this from the constructor. I don't care too much, so maybe I'm nitpicking, but technically, you create 2 instances of the SDK and the second one will override the option of the first one

Reply:
>but technically, you create 2 instances of the SDK

Yeah, I was thinking about that.
It's a bit tricky. The sdk is provided in two ways:
 - `TradingSdk` class
 - set of functions like `postLimitOrder`, `getQuote` and others

Because of that, it's a bit difficult to wire them up together in terms of logging.
I could put `logEnabled` flag to each function, but I fee it a bit overengineering.
I understand that adding `enabled ` property to the `log` function is a hacky way, but this is tradeoff in favor of simplicity.